src/lib/components/DeckGLMap/layers/wells/wellsLayer.js

```diff
@@ -0,0 +1,31 @@
+import { CompositeLayer } from "@deck.gl/core";
+import { GeoJsonLayer } from "@deck.gl/layers";
+
+export default class WellsLayer extends CompositeLayer {
+    renderLayers() {
+        var properties = {
+            id: "wells-shadow",
+            data: this.props.data,
+            opacity: this.props.opacity,
+            pickable: true,
+            pointRadiusUnits: "pixels",
+            lineWidthUnits: "pixels",
+            pointRadiusScale: this.props.pointRadiusScale,
+            lineWidthScale: this.props.lineWidthScale,
+        };
+
+        const shadow = new GeoJsonLayer(properties);
```

Comment:
Not sure how expensive this is, but it could be good to be able to toggle the shadows through a prop, from the json, in case there are a lot of wells and we don't want to render them twice.